Add some missing option doc (#462)
wez/wezterm: docs/config/lua/config/show_update_window.md

# `show_update_window`

> When the automatic update checks detects a new version of Wezterm, this

Reviewer: ```suggestion
When a new version of Wezterm is detected, this
```

or

```
When Wezterm checks for an update and detects a new version, this
```

Because of "checks detects", I had to re-read this part to make sure it isn't a typo. Besides, in the next para, "check_for_updates" is mentioned. So I think we don't need to mention "automatic update checks" here.
